TaskOne/ProjectOne/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+
+namespace ProjectOne
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double x, y, z, result;
+
+            Console.WriteLine("Please enter 3 numbers one by one and press Enter after every number");
+            if (double.TryParse(Console.ReadLine(), out x)) 
```

Comment:
For what we have so much statetements? Could we remove gigantic blocks of if and stop user right after the incorrect input of any of numbers (x, y, or z) ?